### PR TITLE
refactor: rename ChamberProxy to MultiProxy

### PR DESCRIPTION
**Pull request by jyotirmoydotdev**
# Refactoring Updates

- Renamed `ChamberProxy` to `MultiProxy`.
- Updated SPDX-License-Identifier to MIT in all files.
- Moved the emit position of `ProposalExecuted` after the guard.

### CODE REVIEW

File: test/Proxy.t.sol [hunk 1]
```diff
@@ -23,8 +23,8 @@ contract ProxyUpgradeTest is Test {
     IRegistry registry;
     IChamber chamber;
 
-    IChamberProxy registryProxy;
-    IChamberProxy chamberProxy;
+    IChamberProxy multiProxy;
+    IChamberProxy MultiProxy;
```
Comment (xhad) [inline test/Proxy.t.sol]: This is confusing. Please revert back to calling the proxies `registryProxy` and `chamberProxy`

File: test/Proxy.t.sol [hunk 2]
```diff
@@ -38,49 +38,49 @@ contract ProxyUpgradeTest is Test {
         registryProxyAddr = registryDeployer.deploy(address(this));
         chamberProxyAddr = IRegistry(registryProxyAddr).deploy(address(mERC721), address(mERC20));
         
-        chamberProxy = IChamberProxy(chamberProxyAddr);
-        registryProxy = IChamberProxy(registryProxyAddr);
+        MultiProxy = IChamberProxy(chamberProxyAddr);
+        multiProxy = IChamberProxy(registryProxyAddr);
```
Comment (xhad) [inline test/Proxy.t.sol]: revert

**Review by xhad — CHANGES_REQUESTED** [review]

File: test/Proxy.t.sol (hunk 2 above)
Comment (jyotirmoydotdev) [reply test/Proxy.t.sol]: Oky just a min

Comment (xhad) [reply test/Proxy.t.sol]: chamberProxy = IMultiProxy(chamberProxyAddr);
registryProxy = IMultiProxy(registryProxyAddr);

File: test/Proxy.t.sol (hunk 1 above)
Comment (jyotirmoydotdev) [reply test/Proxy.t.sol]: done

File: test/Proxy.t.sol (hunk 2 above)
Comment (jyotirmoydotdev) [reply test/Proxy.t.sol]: Done

**Review by xhad — APPROVED** [review]

**Pull request merged**